frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/where-to-find/where-to-find.component.html

```diff
@@ -77,7 +77,7 @@ <h3>Prohibited areas and restrictions</h3>
 
 <!-- flathead block -->
 <div *ngIf="(forest.forestAbbr)=='flathead'">
```

Comment:
If you do end up changing the code style above, would you mind fixing this one as well?

Reply:
donesky!